al2.pkr.hcl

```diff
@@ -22,6 +25,7 @@ source "amazon-ebs" "al2" {
   }
   ssh_username = "ec2-user"
```

Comment:
There is also a change in the `main` branch here that looked potentially useful:
```
-  ssh_username = "ec2-user"
+  ssh_interface = "public_ip"
+  ssh_username  = "ec2-user"
```

Reply:
Seems to be default, but no harm